updatecli/scripts/updateGomodGoversion.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+## This script updates the go version of a go mod file and prints the resulting content in the stdout
+## Please note that it will NEVER change the go.mod file (works in a temp directory)
+##
+## Expected arguments:
+## - 1: the path to the go.mod file
+## - 2: the new golang version in semver format (major.minor.patch)
+
+set -eux
+
+go_mod_dir="$(dirname "${1}")"
+go_version="${2}"
+new_version="$(echo "${go_version}" | cut -d. -f1,2)"
+tmp_dir="$(mktemp -d)"
+
+## Ensures that there is a golang version installed
+{
+  if ! command -v go
+  then
+    curl --silent --show-error --location --output "${tmp_dir}/go.tgz" \
+      "https://golang.org/dl/go${go_version}.$(uname -s | tr '[:upper:]' '[:lower:]')-amd64.tar.gz"
+    mkdir -p "${tmp_dir}/.bin"
+    tar xzf "${tmp_dir}/go.tgz" -C "${tmp_dir}/.bin"
+    export PATH="${PATH}":"${tmp_dir}"/.bin/go/bin
+  fi
+
+  command -v go
+  cat /etc/*release*
+  ldd $(which go)
+  go version
+} >&2
+
+## Copy go mod's directory to a temp directory an start working from this temp. dir.
```

Comment:
```suggestion
## Copies go mod's directory to a temp directory an start working from this temp. dir.
```

Reply:
Both are fine :) I prefer the imperative form (without the `s`)

Reply:
Me too. Ignore my other suggestions and remove the "s" of the first comment :)

Reply:
are you aware that you are commenting a code I copy/paste from another repository ? ;)

Reply:
i don't remember all code of all repos ^^ It's now on this PR I'm seing stuff to be modified, thus this suggestion :)

Reply:
I'm resolving this conversation so we can focus on the functional part :)